packages/scandipwa/src/component/CheckoutAddressForm/CheckoutAddressForm.component.js

```diff
@@ -30,28 +30,40 @@ export class CheckoutAddressForm extends MyAccountAddressForm {
         onShippingEstimationFieldsChange: () => {}
     };
 
-    onChange = debounce((key, value) => {
-        this.setState(() => ({ [key]: value }));
-    }, UPDATE_STATE_FREQUENCY);
+    onChange = (key, value) => this.setState(() => ({ [key]: value }));
 
     __construct(props) {
         super.__construct(props);
 
         const {
-            address: { region: { region = '' } = {} }
+            shippingFields: {
+                city = '',
+                region_id: regionId = null,
+                region_string: region = '',
+                country_id: countryId = '',
+                postcode = ''
+            }
         } = this.props;
```

Comment:
I'm not sure but github shows all next lines with some strange color, like it's all in the string. Please make double check that this code is okay

Reply:
Checked, it's fine.